EntAppProject/src/main/java/com/jobook/EntAppProject/model/Job.java

```diff
@@ -1,126 +1,124 @@
-package com.jobook.EntAppProject.model;
-
-import java.util.Date;
-
-public class Job {
-	private Integer jobId, customerId, employeeId;
-	private Customer customer;
-	private Employee employee;
-	private String longDesc, shortDesc;
-	private Date startDate, endDate;
-	private Boolean hasBeenReviewed, completed;
-	private Double price;
-	private Review review;
-	
-	public Job(Integer jobId, Integer customerId, Integer employeeId, Customer customer, Employee employee, String longDesc, String shortDesc, Date startDate,
-			Date endDate, Double price) {
-		super();

```

Comment:
As the pull request comment mentions, having `super()` in a class that only extends `Object` by default is unnecessary as that initialization/association is created automatically when the program is run. This change is present in many of the models.